src/uct/api/uct.h

```diff
@@ -1438,8 +1459,8 @@ UCT_INLINE_API unsigned uct_worker_progress(uct_worker_h worker)
  * the data transfer is completed but the target buffer may not be updated yet.
  *
  * @param [in]    iface  Interface to flush communications from.
- * @param [in]    flags  Flags that control completion semantic (currently
- *                        unsupported - set to 0).
+ * @param [in]    flags  Flags that control completion semantic (currently only
+ *                       @ref UCT_FLASH_FLAG_LOCAL is supported).
```

Comment:
Sorry...I had a typo in my suggested text, it should be UCT_FLUSH_FLAG_LOCAL, not UCT_FLASH_FLAG_LOCAL.